storage/postgres/storage.sql

```diff
@@ -4,11 +4,11 @@
 -- ---------------------------------------------
 
 -- Tree Enums
-CREATE TYPE E_TREE_STATE AS ENUM('ACTIVE', 'FROZEN', 'DRAINING');
-CREATE TYPE E_TREE_TYPE AS ENUM('LOG', 'MAP', 'PREORDERED_LOG');
-CREATE TYPE E_HASH_STRATEGY AS ENUM('RFC6962_SHA256', 'TEST_MAP_HASHER', 'OBJECT_RFC6962_SHA256', 'CONIKS_SHA512_256', 'CONIKS_SHA256');
-CREATE TYPE E_HASH_ALGORITHM AS ENUM('SHA256');
-CREATE TYPE E_SIGNATURE_ALGORITHM AS ENUM('ECDSA', 'RSA');
+CREATE TYPE E_TREE_STATE AS ENUM('ACTIVE', 'FROZEN', 'DRAINING');--end
+CREATE TYPE E_TREE_TYPE AS ENUM('LOG', 'MAP', 'PREORDERED_LOG');--end
+CREATE TYPE E_HASH_STRATEGY AS ENUM('RFC6962_SHA256', 'TEST_MAP_HASHER', 'OBJECT_RFC6962_SHA256', 'CONIKS_SHA512_256', 'CONIKS_SHA256');--end
+CREATE TYPE E_HASH_ALGORITHM AS ENUM('SHA256');--end
+CREATE TYPE E_SIGNATURE_ALGORITHM AS ENUM('ECDSA', 'RSA');--end
```

Comment:
Great question!  The testdb code splits on semicolons when executing storage.sql.  Since I've added some functions at the bottom of the file, they don't get run properly because the functions have internal semicolons which aren't intended to be separate operations.  I added the --end comment to signify these are standalone sql commands and should be executed as a block.  They don't affect other uses of the storage.sql file and allow the tests to run.